src/llmq/params.h

```diff
@@ -440,6 +441,33 @@ static constexpr std::array<LLMQParams, 13> available_llmqs = {
         .recoveryMembers = 50,
     },
 
+    /**
+     * llmq_25_67
+     * This quorum is deployed on Testnet and requires
+     * 25 participants
+     *
+     * Used by Dash Platform
+     */
+    LLMQParams{
+        .type = LLMQType::LLMQ_25_67,
+        .name = "llmq_25_67",
+        .useRotation = false,
+        .size = 25,
+        .minSize = 22,
+        .threshold = 17,
+
+        .dkgInterval = 24, // one DKG per hour
+        .dkgPhaseBlocks = 2,
+        .dkgMiningWindowStart = 10, // dkgPhaseBlocks * 5 = after finalization
+        .dkgMiningWindowEnd = 18,
+        .dkgBadVotesThreshold = 80,
```

Comment:
20-22, not sure how we pick this number exactly

Reply:
@UdjinM6 You are referring to `minSize`? And/or the size itself ?
Note: Just noticed that `dkgBadVotesThreshold` is wrong

Reply:
I was suggesting values for `dkgBadVotesThreshold` :)